samples/NativeLibrary/LoadLibrary.c

```diff
@@ -93,3 +108,35 @@ char *callSumStringFunc(char *path, char *funcName, char *firstString, char *sec
     // See https://github.com/dotnet/corert/issues/7887
     return result;
 }
+
+char *callMergeStrArray(char *funcName, char **array, int count, void **handlePtr)
+{
+    char *returnStr = malloc(64 * sizeof(char));
+    // Declare a typedef
+    typedef void (*myFunc)();
+
+    // Import Symbol named funcName
+    myFunc MyImport = (myFunc)symLoad(*handlePtr, funcName);
+
+    // The C# function will return a pointer
+    MyImport(returnStr, array, count);
+
+    // CoreRT libraries do not support unloading
+    // See https://github.com/dotnet/corert/issues/7887
+    return returnStr;
+}
+
+void *callPopulateStruct(char *funcName, void *structure, void **handlePtr)
+{
+    // Declare a typedef
+    typedef void (*myFunc)();
+
+    // Import Symbol named funcName
+    myFunc MyImport = (myFunc)symLoad(*handlePtr, funcName);
+
+    // The C# function will return a pointer
+    MyImport(structure);
+
+    // CoreRT libraries do not support unloading
```

Comment:
There comments are not relevant anymore now that these methods do not call LoadLibrary.